benches/graphemes.rs

```diff
@@ -1,13 +1,8 @@
 mod common;
 
 use common::{LARGE, MEDIUM, SMALL, TINY};
-use criterion::{criterion_group, criterion_main, Criterion};
-use crop::{iter::Graphemes, Rope};
-
-const TINY: &str = include_str!("../tests/common/tiny.txt");
-const SMALL: &str = include_str!("../tests/common/small.txt");
-const MEDIUM: &str = include_str!("../tests/common/medium.txt");
-const LARGE: &str = include_str!("../tests/common/large.txt");
+use criterion::{Criterion, criterion_group, criterion_main};
```

Comment:
Looks like `rustfmt` also re-ordered the imports here (I think it's the Rust 2024 edition formatting style to put types first?). That's why this line was changed.

Reply:
It is, but we're still on `edition = "2021"` so I'm not sure why rustfmt would format it that way. It may be something specific to your setup, `cargo fmt` is a no-op on my machine.

Let's remove this line from the diff so that CI passes, I'll update the project to the 2024 edition in a follow-up PR.

Reply:
It is, but we're still on `edition = "2021"` so I'm not sure why rustfmt would format it that way. It may be something specific to your setup, `cargo fmt` is a no-op on my machine.

Let's remove this line from the diff so that CI passes, I'll update the project to the 2024 edition in a follow-up PR.